telemetry/otel-collector-config.yaml

```diff
@@ -1,9 +1,29 @@
+# https://opentelemetry.io/docs/collector/configuration/
+
+# How data gets into the Collector
 receivers:
+  # Data sources: traces, metrics, logs
   otlp:
     protocols:
+      http:
+        endpoint: "otel-collector:4318"
       grpc:
+        endpoint: "otel-collector:4317"
+  # Data sources: metrics
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: openfga/openfga
+          scrape_interval: 1s
+          static_configs:
+            - targets: [ openfga:2112 ]
+              labels:
+                service: openfga
 
+# How you send data to one or more backends. Configuring an exporter does not enable it.
 exporters:
+  # Data sources: traces
+  # TODO will be deprecated July 2023 https://github.com/open-telemetry/opentelemetry-specification/pull/2858
```

Comment:
Please add this as an issue instead of a comment. It'll be forgotten if we leave it here as a comment.

Reply:
https://github.com/openfga/openfga/issues/609